cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DistributedPreviewManager.java

```diff
@@ -217,7 +216,7 @@ public void run() {
             String localhost = InetAddress.getLoopbackAddress().getHostName();
             twillPreparer = twillPreparer.withEnv(PreviewRunnerTwillRunnable.class.getSimpleName(),
                 ImmutableMap.of(
-                    ArtifactLocalizer.GCE_METADATA_HOST_ENV_VAR,
+                    Constants.Preview.GCE_METADATA_HOST_ENV_VAR,
```

Comment:
nit: Any particular reason we duplicated this environment variable value?

Reply:
This env variable is set iin `taskworkertwillrunnable` and `previewrunnertwillrunnable` and not `artifactlocalizertwillrunnable` so wanted to reflect what it does.